test/controllers/images_controller_test.rb

```diff
@@ -0,0 +1,48 @@
+require "test_helper"
+
+class ImagesControllerTest < ActionDispatch::IntegrationTest
```

Comment:
I like how you've extracted images into their own class, meaning you can update, destroy or add images without altering the overall post